pkg/osvscanner/osvscanner.go

```diff
@@ -967,7 +967,17 @@ func filterIgnoredPackages(r reporter.Reporter, packages []scannedPackage, confi
 	out := make([]scannedPackage, 0, len(packages))
 	for _, p := range packages {
 		configToUse := configManager.Get(r, p.Source.Path)
-		if ignore, ignoreLine := configToUse.ShouldIgnorePackageVersion(p.Name, p.Version, string(p.Ecosystem)); ignore {
+		if ignore, ignoreLine := configToUse.ShouldIgnorePackage(
```

Comment:
nit: Separate this out from the if statement as it is quite long, and just have it on it's own line.

Reply:
done - I went with keeping the function in the if and using a variable for the struct as I think that's a bit more correct: in the long-run, we expect to be passing a whole "thing" to this function (like what we're already doing in `vulnerability_result.go`), so it's the `models.PackageVuln` part that's a temp workaround